Cargo.toml

```diff
@@ -3,7 +3,9 @@ name = "voprf"
 version = "0.1.0"
 description = "An implementation of a verifiable oblivious pseudorandom function (VOPRF)"
 authors = ["Kevin Lewi <klewi@fb.com>"]
-categories = ["no-std"]
+repository = "https://github.com/novifinancial/voprf/"
+categories = ["no-std", "algorithms", "cryptography"]
+keywords = ["oprf"]
```

Comment:
I hope this is appropriate. More keywords might be good, but I will leave that to you.

Reply:
Thank you, this looks good for now!